canvas_modules/common-canvas/src/common-properties/controls/slider/slider.jsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2017-2023 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { Slider } from "carbon-components-react";
+import { connect } from "react-redux";
+import classNames from "classnames";
+import { v4 as uuid4 } from "uuid";
+
+import * as ControlUtils from "../../util/control-utils";
+import ValidationMessage from "../../components/validation-message";
+import { STATES } from "../../constants/constants";
+
+
+class SliderControl extends React.Component {
+	constructor(props) {
+		super(props);
+		this.handleChange = this.handleChange.bind(this);
+		this.uuid = uuid4();
+		this.id = ControlUtils.getControlId(props.propertyId, this.uuid);
+		this.state = {
+			value: props.value,
+			min: props.control.min_value,
+			max: props.control.max_value,
+			step: props.control.increment
+		};
+	}
+
+	handleChange(e) {
+		this.props.controller.updatePropertyValue(this.props.propertyId, e.value);
+	}
+
+	render() {
+		const minLabel = this.props.controller.getResource(`${this.props.control.name}.min.label`, null);
+		const maxLabel = this.props.controller.getResource(`${this.props.control.name}.max.label`, null);
+		return (
+			<div className={classNames("properties-slider ", { "hide": this.props.state === STATES.HIDDEN })}
+				data-id={ControlUtils.getDataId(this.props.propertyId)}
+			>
+				<Slider
+					id={this.id}
+					value={this.state.value}
+					min={this.state.min}
```

Comment:
What happens in min, max, step aren't defined?  Shouldn't we default them in the code?